wiredep.js

```diff
@@ -85,7 +85,8 @@ function mergeFileTypesWithDefaults(optsFileTypes) {
   var fileTypes = $._.clone(fileTypesDefault, true);
 
   $._(optsFileTypes).each(function (fileTypeConfig, fileType) {
-    fileTypes[fileType] = fileTypes[fileType] || {};
+    // fallback to the default type for all html-like extensions (php, twig, hbs, etc)
+    fileTypes[fileType] = fileTypes[fileType] || fileTypes['default'];
```

Comment:
Can we just make this change and skip the extra tests? Also, somehow the permissions of this file changed from 644 -> 755. Can you bring that back?